src/jitlog/reader.lua

```diff
@@ -1,6 +1,7 @@
 local ffi = require"ffi"
 require("table.new")
 local format = string.format
+local tinsert = table.insert
```

Comment:
Is this worth making a `local` given it's only used once?

Reply:
OK, it's used twice but still...

Reply:
i did mean to go through and update all the other uses after with another commit

Reply:
OK, let's do that, then we're getting close to being able to merge this.